FinSearchAPI/BusinessLogic/FindologicArticleFactory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace FinSearchAPI\BusinessLogic;
+
+use FinSearchAPI\BusinessLogic\Models\FindologicArticleModel;
+use Shopware\Models\Article\Article;
+
+class FindologicArticleFactory
+{
+    /**
+     * FindologicArticleModel constructor.
+     *
+     * @param Article $shopwareArticle
+     * @param string  $shopKey
+     * @param array   $allUserGroups
+     * @param array   $salesFrequency
+     *
+     * @throws \Exception
+     */
+    public function createFindologicArticle(Article $shopwareArticle, $shopKey, array $allUserGroups, array $salesFrequency)
```

Comment:
Since all the info regarding FINDOLOGIC and articles is already in the class name, I'd propose to rename this method to just `create`. So you will end up with something like
```
$findologicArticleFactory->create(...);
```

Reply:
Done. Also updated the docblock info about the method